app/controllers/users_controller.rb

```diff
@@ -1,5 +1,8 @@
 class UsersController < ApplicationController
   def new
+    if params[:message]
```

Comment:
In the view.  I don't know of a better way to do this.  It's what Jorge suggested.